File: c/ejercicios_todocode/intercambio_de_valores/intercambio_de_valores.c
#include <stdio.h>

int main(void)
{
    int n1 = 35, n2 = 20, aux = 0;

    printf("\nn1 vale %i y n2 vale %i", n1, n2);

    aux = n1;
    n1 = n2;
    n2 = aux;

    printf("\nn1 vale %i y n2 vale %i\n", n1, n2);

    return 0;
}

File: python/ejercicios_todocode/intercambio_de_valores/intercambio_de_valores.py
"""
2) Realizar un programa que permita el intercambio de valores entre dos variables.
Por ejmplo: Si una variable numero vale 35, y una variable numero2 vale 20,
pase a valer 35. Una ves realizado el cambio a mostrar el resultado en la pantalla.
"""

n1 = 35
n2 = 20

print(f"\nn1 vale {n1} y n2 vale {n2}")

n1, n2 = n2, n1

print(f"\nn1 vale {n1} y n2 vale {n2}\n")